starknet_py/common.py

```diff
@@ -40,3 +66,17 @@ def int_from_bytes(
     Converts the given bytes object (parsed according to the given byte order) to an integer.
     """
     return int.from_bytes(value, byteorder=byte_order, signed=signed)
+
+
+def get_storage_var_address(var_name: str, *args) -> int:
+    """
+    Returns the storage address of a StarkNet storage variable given its name and arguments.
+    """
+    res = starknet_keccak(var_name.encode("ascii"))
+
+    for arg in args:
+        if not isinstance(arg, int):
+            raise TypeError(f"Expected arguments to be integers. Found: {arg}.")
```

Comment:
I think we can just type args properly.

Reply:
What exactly do you mean by typing args properly?

Reply:
This: https://stackoverflow.com/a/37032111. Currently `get_storage_var_address` accepts anything, so it has to check types in the runtime.